tests/csm/rest/test_capacity_usage.py

```diff
@@ -94,11 +96,15 @@ def setup_class(cls):
                                  password=CMN_CFG["nodes"][0]["password"])
 
         cls.log.debug("Node object list : %s", cls.nd_obj)
-        cls.restore_method = RESTORE_SCALE_REPLICAS
+        cls.restore_method = None
```

Comment:
if this is not getting used, can we remove ?

Reply:
It is being used further as observed in the file

Reply:
yes we should remove then

Reply:
will need to remove as we have removed restore_pod call from teardowm
maybe next time when we revisit these tests